rules/falco_rules.yaml

```diff
@@ -1859,6 +1859,7 @@
     - public.ecr.aws/falcosecurity/falco-no-driver
 
 # These container images are allowed to run with --privileged and full set of capabilities
+# TODO: Remove k8s.gcr.io reference after 01/Dec/2023
```

Comment:
@leogr @LucaGuerra  added this date to we remove the k8s reference after December 
the k8s registry will be frozen on 3rd of April 2023 (https://kubernetes.io/blog/2023/02/06/k8s-gcr-io-freeze-announcement/) so we give some extra time for people to update the clusters.
wdyt?